### ISSUE TITLE
Admin opt out of emails in projects

### ISSUE THREAD

**Issue by fh948h9** [opencollective/opencollective#7243]
It would be helpful if project admins were given an option to opt out of email notifications of individual projects they’re an admin of. 

This is so that admins can stay opted into email notifications for some projects, but not for others. 

Sometimes an admin doesn’t need to receive email notifications of a project they’re an admin of. It can cause unnecessary clutter of the email inbox. And admins can still meaningfully fulfill their share of co-administering a project without opting in to receive emails at all times. 

One solution I thought of is filtering the email inbox to leave out certain project emails but that’s pretty complicated for a lot of people to do.

**Comment by shannondwray** [comment]
Hello! You can do this here: https://docs.opencollective.com/help/product/notifications Please let us know if you have any further questions :)

**Comment by fh948h9** [comment]
Based on what I read in https://docs.opencollective.com/help/product/notifications the feature I'm asking for does not exist. Perhaps I read it wrong. 

I went to the collective's settings at https://opencollective.com/dashboard/var/notifications/collectives , and I don't c the feature I'm writing about, even when I look into the "advanced settings". 

![image](https://github.com/opencollective/opencollective/assets/100950965/a21be718-3968-49e8-9713-a1b85a8c4404)

**Comment by fh948h9** [comment]
I don't see such notification settings in https://opencollective.com/dashboard/tenants-united-south-chicago/advanced either.

**Comment by fh948h9** [comment]
When I look at an email notification from an expense I only c the option to opt out of expense settings for the collective, not the individual project. 
![image](https://github.com/opencollective/opencollective/assets/100950965/268145e4-c4d1-4957-83e3-7a38c6657988)

**Comment by fh948h9** [comment]
And i don't see such options in any project's settings ("dashboard"), eg, https://opencollective.com/dashboard/280-rf-2o/

**Comment by Betree** [comment]
That's right; we currently don't support opting out of emails at the projects/events (children) level. Our data model should be mostly ready to handle this, but figuring out the proper UI will be more challenging.

**Comment by fh948h9** [comment]
It would be really good for the opt out option to be available in the email itself if ever y’all were able to figure it out.
